nats/subsz.yml

```diff
@@ -0,0 +1,60 @@
+type: json
+data:
+  url: /subsz
+  server: ${NATS_HOSTPORT}
+
+observation:
+  - name: subsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: subscriptions
+        source: num_subscriptions
+        type: long_gauge
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: inserts
+        source: num_inserts
+        type: long_gauge
+        label: inserts
+        description: The number of insert operations in subscriptions list
+
+      - name: removes
+        source: num_removes
+        type: long_gauge
+        label: removes
+        description: The number of remove operations in subscriptions list
+
+      - name: matches
+        source: num_matches
+        type: long_gauge
+        label: matches
+        description: The number of times a match is found for a subscription
+
+      - name: cache
```

Comment:
Rename to `cache.size`? If yes, also rename the label accordingly.

Reply:
OK, i change to cache.size